release/air_tests/air_benchmarks/workloads/benchmark_util.py

```diff
@@ -75,3 +75,43 @@ def run_commands_with_resources(
         ).remote(cmd=cmd)
         futures.append(future)
     return ray.get(futures)
+
+
+@ray.remote
+class CommandRunner:
+    def run_command(self, cmd: str):
+        return subprocess.check_call(cmd)
+
+    def run_fn(self, fn: Callable, *args, **kwargs):
+        return fn(*args, **kwargs)
+
+
+def create_actors_with_resources(
+    num_actors: int, resources: Dict[str, Union[float, int]]
+) -> List[ray.actor.ActorHandle]:
+    num_cpus = resources.pop("CPU", 1)
+    num_gpus = resources.pop("GPU", 0)
+
+    return [
+        CommandRunner.options(
+            num_cpus=num_cpus, num_gpus=num_gpus, resources=resources
+        ).remote()
+        for _ in range(num_actors)
+    ]
+
+
+def run_commands_on_actors(actors: List[ray.actor.ActorHandle], cmds: List[List[str]]):
```

Comment:
just wondering, why do you need this instead of run_command_on_all_nodes? It seems that both are being used for the same purpose (running a script on each node in the cluster).

Reply:
Also create_actors makes no guarantee on the placement of the actors right?